File: Example.Tests/CalculatorTests.fs
namespace Example

open Example
open Mutannot
open Xunit

type CalculatorTests() =
    [<Fact>]
    [<ShouldCatch("""
    diff --git a/Example/Calculator.fs b/Example/Calculator.fs
    index 6f0c515..030e391 100644
    --- a/Example/Calculator.fs
    +++ b/Example/Calculator.fs
    @@ -1,4 +1,4 @@
     namespace Example

     module Calculator =
    -    let addOne value = value + 1
    +    let addOne value = value - 1
    """)>]
    member _.AddOne_increments() = Assert.Equal(42, Calculator.addOne 41)

File: Example.Mtp.Tests/CalculatorTests.cs
using Mutannot.Annotations;
using Xunit;

namespace Example.Mtp;

[ShouldCatch("""
--- a/Example.CSharp/Calculator.cs
+++ b/Example.CSharp/Calculator.cs
@@ -1,6 +1,6 @@
 namespace Example;

 public static class Calculator
 {
-    public static int Add(int x, int y) => x + y;
+    public static int Add(int x, int y) => x - y;
 }
""")]
public class CalculatorTests
{
    [Fact]
    public void Add_Returns_Sum()
    {
        Assert.Equal(5, Calculator.Add(2, 3));
    }
}
